tests/mocha/navigation_test.js

```diff
@@ -367,6 +367,9 @@ suite('Navigation', function() {
         }
       };
     });
+    teardown(function() {
+      delete Blockly.Blocks['basic_block'];
```

Comment:
Add: `this.workspace.dispose();`

The teardown should have been doing it before, but wasn't.

Reply:
Done